api/src/main/java/marquez/api/EventResource.java

```diff
@@ -0,0 +1,62 @@
+package marquez.api;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+import com.codahale.metrics.annotation.ExceptionMetered;
+import com.codahale.metrics.annotation.ResponseMetered;
+import com.codahale.metrics.annotation.Timed;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import javax.validation.constraints.Min;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+import lombok.NonNull;
+import lombok.Value;
+import marquez.service.ServiceFactory;
+import marquez.service.models.RawLineageEvent;
+
+@Path("/api/v1")
+public class EventResource extends BaseResource {
+  public EventResource(@NonNull final ServiceFactory serviceFactory) {
+    super(serviceFactory);
+  }
+
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @GET
+  @Path("/events")
```

Comment:
I would use the endpoint `GET` `/lineage_events` to be more specific, and less generic. We'll also be adding a `GET` `/lifecycle_events` endpoint that will capture events like:

* A **new** dataset / job version has created
* A dataset has been **deleted**
* A job has transited to `COMPLETED` at time **X**

See https://github.com/MarquezProject/marquez/issues/1973

Reply:
@wslulciuc maybe `/events/lineage` and `/events/lifecycle`? Not sure which is less confusing between `lineage` and `lifecycle`.

Reply:
@mobuchowski: actually, my initial thought was to use (and the more I think about it, we should):

```
GET /events/lineage
GET /events/lifecycle
```

I can't recall why I convinced myself `/lifecycle_events` was a better alternative. That said, I think using `/events/*` provides callers with a more intuitive API to query events by type; we could also define `GET` `/events` with a `type` param to include events of a particular type. But, the response objects differ enough where it would warrant separate endpoints as you suggested 👍